packages/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -142,6 +142,11 @@ export interface DataGridPropsWithDefaultValues<R extends GridValidRowModel = an
    * @default true
    */
   rowSelection: boolean;
+  /**
+   * The milliseconds debounce delay for resizing the grid.
+   * @default 60
+   */
+  resizeDebounceMs: number;
```

Comment:
If we use throttle instead of debounce, shouldn't we call this one `resizeThrottleMs`?

Reply:
Sure, only saw other "debounce" props, used the same term to avoid adding a new term. The two operations are frequently mixed.